sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -99,7 +101,9 @@ public UserDto selectActiveUserByLogin(DbSession session, String login) {
    * if list of logins is empty, without any db round trips.
    */
   public List<UserDto> selectByLogins(DbSession session, Collection<String> logins) {
-    return DatabaseUtils.executeLargeInputs(logins, new SelectByLogins(mapper(session)));
+    return executeLargeInputs(
+      logins,
+      partitionOfLogins -> session.getMapper(UserMapper.class).selectByLogins(partitionOfLogins));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)